client/app/app.js

```diff
@@ -0,0 +1,37 @@
+var map;
+function initMap() {
```

Comment:
Hey Alice, looks great but maybe we could do 'var initMap = function()' instead?